test-two/src/index.js

```diff
@@ -4,23 +4,29 @@ import { getUsersForOrganisation } from "./services/github";
 const baseUrl = "https://api.github.com/";
 const organisationId = "google";
 const gitHubApiAuthToken = ""; // add your GitHub API OAuth key here to increase request limit
+var pageNumber;
 
 export const go = async () => {
-  try {
-    const users = await getUsersForOrganisation(
-      baseUrl,
-      gitHubApiAuthToken,
-      organisationId
-    );
 
-    const output = users.reduce((result, user) => {
-      return `${result}Username: ${user.login}\n`;
-    }, "");
+    for (pageNumber = 1; pageNumber <= 23; pageNumber++) {
```

Comment:
Can you think of a way to improve the performance of this for loop?

Reply:
Yes, I could of moved the try and catch block out of the loop and had the results sorted in an array and have the them displayed after the loop.